pkg/podman/inspect.go

```diff
@@ -34,8 +34,13 @@ func (o *PodmanCli) PodInspect(podname string) (PodInspectData, error) {
 		return PodInspectData{}, err
 	}
 
-	var result PodInspectData
-
+	var result []PodInspectData
 	err = json.Unmarshal(out, &result)
-	return result, err
+	if err != nil {
+		var singleResult PodInspectData
+		err = json.Unmarshal(out, &singleResult)
+		return singleResult, err
+	}
+
+	return result[0], err
```

Comment:
```suggestion
	if len(result) == 0 {
		return PodInspectData{}, nil
	}
	return result[0], nil
```

Just to be safe and avoid panicking if the list is empty for some reason.